code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -736,6 +736,12 @@
 	eye_icon_state = "jelleyes"
 	icon_state = "eyeballs-jelly"
 
+/obj/item/organ/internal/eyes/shadekin
+	name = "shadekin eyes"
+	desc = "These eyes are massive, and feel warm to the touch. The shadekin that's missing these is probably feeling very queasy."
+	eye_icon_state = "shadekin_eyes"
+	icon_state = "eyeballs-moth"	//i'm too lazy to give them their own sprite
+
```

Comment:
Move this to the modular zubber folder, and override the icon to lead to a modular zubber icon file

Reply:
i have no idea how i would do that without bricking every other eye sprite

Reply:
```
/obj/item/organ/internal/eyes/shadekin
	name = "shadekin eyes"
	desc = "These eyes are massive, and feel warm to the touch. The shadekin that's missing these is probably feeling very queasy."
	eye_icon_state = "shadekin_eyes"
	icon = 'modular_zubbers/filepath/iconfile.dmi'
	icon_state = "eyeballs-moth"
```

Reply:
unfortunately the code that handles rendering the eye sprites (like, the mob ones, not the item ones) would love to disagree with that
![imagen](https://github.com/Bubberstation/Bubberstation/assets/3855300/2d109313-a2dc-456b-a1be-5bc14c43ca26)

Reply:
Ok yea I read through it again and yea I done fucked up. Anyhow this will most likely not last forever as any upstream changes to the dmi will override it

Reply:
![image](https://github.com/Bubberstation/Bubberstation/assets/49160555/4928b408-d330-4ae6-a993-e3224872a2ca)
Got it @Kraseo 
Give the species its own eyes_icon